storages/backends/google_cloud.py

```diff
@@ -0,0 +1,62 @@
+import base64
+import json
+from datetime import datetime
+from typing import AnyStr, Optional
+
+from google.cloud import storage
+
+from storages.backends.base import Storage
+from storages.exceptions import ImproperlyConfiguredError
+
+
+class GoogleCloudStorage(Storage):
+    _SERVICE_NAME = "google_cloud"
+
+    def __init__(
+        self,
+        google_cloud_credentials: str,
+        bucket_name: str,
+    ):
+        if not google_cloud_credentials:
+            raise ImproperlyConfiguredError(
+                name="credentials_path", value=google_cloud_credentials
+            )
+        if not bucket_name:
+            raise ImproperlyConfiguredError(
+                name="bucket_name", value=bucket_name
+            )
+
+        self._client = storage.Client.from_service_account_info(
+            json.loads(base64.b64decode(google_cloud_credentials))
+        )
+        self._bucket_name = bucket_name
+        self._bucket = self._client.get_bucket(self._bucket_name)
+
+    def read(self, name: str, mode: str = "r") -> AnyStr:
+        blob = self._bucket.blob(name)
+        return blob.download_as_bytes()
+
+    def write(self, name: str, content: AnyStr, mode: str = "a"):
+        self._bucket.blob(name).upload_from_string(content)
+
+    def delete(self, name: str):
+        self._bucket.blob(name).delete()
+
+    def exists(self, name: str) -> bool:
+        return self._bucket.blob(name).exists()
+
+    def size(self, name: str) -> int:
+        return self._bucket.get_blob(name).size
+
+    def get_created_time(self, name: str) -> Optional[datetime]:
+        blob = self._bucket.get_blob(name)
+        return blob and blob.time_created
```

Comment:
Was the intention here to verify if the `blob` is not `None` and if not then return `blob.time_created` or return `None` otherwise? If so, then I strongly suggest throwing an exception here when `blob` does not exist and make this method return `datetime` object always.